### DISCUSSION TITLE
Confusion while masking

### DISCUSSION THREAD

**Discussion by AkbarAC** [AstroBridge/BDLensing#175] [Q&A] [answered]
Hi, I am getting hold of masking. The screenshots attached here show two masked plots. Which one is correct? As you can see two blobs around the center and both of them are almost equally prominent. Should I mask one or is there something else I need to consider while running pso due to this? TIA 

![image](https://github.com/ajshajib/BDLensing/assets/130357455/6e8246fd-b6b4-4d04-bb59-c26bf73dbc90)
![mask](https://github.com/ajshajib/BDLensing/assets/130357455/ea6677a7-37dc-4e1f-a3ba-c507866632dc)
![mask2](https://github.com/ajshajib/BDLensing/assets/130357455/aaec492d-8927-4b8e-98bc-125709e16b6a)

**Comment by ajshajib** [comment, marked as answer]
Look at @rafee-adnan's system, which has similarly two bright galaxies. His model doesn't mask either of them and includes both in the mass and light models. But you can mask the tiny galaxy.

**Reply by AkbarAC** [reply]
Got it, thanks. I have decided to mask the tiny galaxy as it gave me a better chi^2 value when masked.
